kingpin/actors/rightscale/server_array.py

```diff
@@ -978,24 +985,27 @@ def _wait_until_healthy(self, array, sleep=60):
         # Get the current min_count setting from the ServerArray object, or get
         # the min_count from the count number supplied to the actor (if it
         # was).
-        min_count = int(self._options.get('count', False))
+        min_count = float(self.option('count'))
+        wait_pct = float(self.option('wait_pct'))
         if not min_count:
             min_count = int(array.soul['elasticity_params']
                             ['bounds']['min_count'])
 
+        min_wait = int(math.ceil(min_count * (wait_pct / 100.0)))
```

Comment:
Success_pct maybe? Could be success count too. I think framing it in terms
of the success criteria for the user might be clearest because from the
users point of view the target is the min count.
On Thu, Jun 2, 2016 at 11:31 PM Mikhail Simin <notifications@github.com>
wrote:

> In kingpin/actors/rightscale/server_array.py
> <https://github.com/Nextdoor/kingpin/pull/376#discussion_r65661750>:
>
> >          if not min_count:
> >              min_count = int(array.soul['elasticity_params']
> >                              ['bounds']['min_count'])
> >
> > +        min_wait = int(math.ceil(min_count * (wait_pct / 100.0)))
>
> Thanks! I was struggling to find it. Hate the wait term. Maybe rename the
> option to target_pct also?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nextdoor/kingpin/pull/376/files/0e723fb6ac2686255bbfe5f4648f741ebb0038bd#r65661750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxZ_t4WpLZC4hkfgvmGTyWm115shvmcks5qH8oxgaJpZM4Is9Yd>
> .
>